### PR TITLE
add SPIKE Essential hub

### PR DESCRIPTION
**Pull request by NStrijbosch**

### CODE REVIEW

**Comment by dlech** [comment]
Did you read this from the advertising data and/or the hub properties?

**Comment by NStrijbosch** [comment]
By asking the hub for a update on its HubProperty hub_kind:

```
>>> HubPropertyRequestUpdate(HubProperty(11))
[22:32:46.022] INFO: sending: HubPropertyRequestUpdate(<HubProperty.HUB_KIND: 11>)
[22:32:46.230] INFO: received: HubPropertyUpdate(<HubProperty.HUB_KIND: 11>, <HubKind.TECHNIC_SMALL: 131>)
```

I am surprised by the missing 0x82 too..

File: pybricksdev/ble/lwp3/bytecodes.py [hunk 1]
```diff
@@ -194,6 +194,9 @@ class HubKind(IntEnum):
 
     TECHNIC_LARGE = 0x81
     """LEGO SPIKE Prime Hub and MINDSTORMS Robot Inventor hubs."""
+
```
Comment (dlech) [inline pybricksdev/ble/lwp3/bytecodes.py:197]: Looks like there was some extra whitespace here.

```suggestion

```

**Comment by dlech** [comment]
> I am surprised by the missing 0x82 too..

Maybe it is supposed to be Robot Inventor (eventually)?

**Comment by dlech** [comment]
Merged, thanks!

**Pull request merged**